### ISSUE TITLE
annoying warning

### ISSUE THREAD

**Issue by beardymcjohnface** [YiqunChen1999/attrmap#2]
Thank you for creating this nice package. 

I'm not sure why, but initialising more than one level deep throws an annoying warning message:

```shell
$ pip show attrmap
Name: attrmap
Version: 0.0.6
...

$ python --version
Python 3.8.10

$ python
>>> import attrmap as ap
>>> d = ap.AttrMap()
>>> d.yeet.ya = 'yee'
.../python3.8/site-packages/attrmap/attrmap.py:270: UserWarning: To reduce 'problematic' attributes, `read_only` is deprecated, use `attrmap.utils.is_read_only` instead.
  warn("To reduce 'problematic' attributes, `read_only` is deprecated, "
```

**Comment by YiqunChen1999** [comment]
Thanks for pointing out this issue. I'll fix it as soon as possible.

**Comment by YiqunChen1999** [comment]
Hi, this issue has been fixed in v0.0.7. Please have a try. Please let me know if you find other bugs.

**Comment by beardymcjohnface** [comment]
Yes, works perfectly. Thanks!
